docs/sources/panels/visualizations/geomap.md

```diff
@@ -8,4 +8,123 @@ weight = 600
 
 # Geomap
 
-The Geomap panel visualization allows you to view ...
\ No newline at end of file
+{{< figure src="/static/img/docs/geomap-panel/geomap_example.png" max-width="1200px" caption="Geomap panel" >}}
+
+The Geomap panel visualization allows you to view and customize the world map using geospatial data in respect to geographical locations. Various overlay styles and map view settings can be configured to easily focus on the important location-based characteristics of the data.
```

Comment:
Suggestion: `The Geomap panel visualization allows you to view and customize the world map using geospatial data.` (Delete `in respect to geographical locations` - might be redundant.)